balancer/rls/internal/builder.go

```diff
@@ -43,3 +47,8 @@ func (*rlsBB) Build(cc balancer.ClientConn, opts balancer.BuildOptions) balancer
 	// TODO(easwars): Fix this once the LB policy implementation is pulled in.
 	return &rlsBalancer{}
 }
+
+func (*rlsBB) ParseConfig(c json.RawMessage) (serviceconfig.LoadBalancingConfig, error) {
```

Comment:
Not a major deal but we tend to use non-pointer receivers for balancer builders (and register the struct instead of a pointer to a struct instance).  AIUI it's slightly more optimized that way due to memory allocations.

Reply:
Done.